widgets/WebComponents/AvatarIcon/AvatarIcon.bbj

```diff
@@ -0,0 +1,46 @@
+use ::WebKit/widgets/WebComponents/WebComponents.bbj::WebComponents
+use ::WebKit/util/ClientUtil.bbj::ClientUtil
+
+class public AvatarIcon extends WebComponents
```

Comment:
This class should follow the BBjWidget pattern and not use some custom methods to use the widget (drawAvatarIcon)